packages/fast-usdc/src/cli/lp-commands.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @import {Command} from 'commander';
+ * @import {OfferSpec} from '@agoric/smart-wallet/src/offers.js';
+ * @import {ExecuteOfferAction} from '@agoric/smart-wallet/src/smartWallet.js';
+ */
+
+import { fetchEnvNetworkConfig, makeVstorageKit } from '@agoric/client-utils';
+import { Nat } from '@endo/nat';
+import { InvalidArgumentError } from 'commander';
+import { outputActionAndHint } from './bridge-action.js';
+
+/** @param {string} arg */
+const parseNat = arg => {
+  try {
+    const n = Nat(BigInt(arg));
+    return n;
+  } catch {
+    throw new InvalidArgumentError('Not a number');
+  }
+};
+
+/**
+ * @param {Command} program
+ * @param {{
+ *   fetch?: Window['fetch'];
+ *   vstorageKit?: Awaited<ReturnType<typeof makeVstorageKit>>;
```

Comment:
These can't both be omitted, right?

```suggestion
 * @param {(
 *  { vstorageKit?: Awaited<ReturnType<typeof makeVstorageKit>> }
 * | { fetch: Window['fetch'] })
```

Reply:
This doesn't work as expected, I get a type error when trying to access either property when I do that.

Reply:
ah. right; you'd have to test for `'fetch' in opts`. oh well.